subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/AndroidSantaTrackerSmokeTest.groovy

```diff
@@ -84,11 +107,17 @@ class AndroidCachingSmokeTest extends AbstractSmokeTest {
                 }
             }
         """ + settingsFile.text
+    }
 
-        return git
+    private BuildResult buildLocation(File projectDir) {
+        runner("assembleDebug", "--scan", "-D" + DefaultGradleUserHomeScopeServiceRegistry.REUSE_USER_HOME_SERVICES + "=false")
```

Comment:
Please add a comment about why this is needed.

Reply:
I removed the flag instead. When killing the daemons, all locks are gone.